controllers/tobiko_controller.go

```diff
@@ -18,6 +18,7 @@ package controllers
 
 import (
 	"context"
+    "strconv"
```

Comment:
super small nit ... The tabs :D ... I understand.

Reply:
no, it's fine. I'm fixing it.
Thanks for the catch :)